### DISCUSSION TITLE
建议Exercises使用ipynb而不是markdown

### DISCUSSION THREAD

**Discussion by hbsun2113** [mlc-ai/mlc-zh#36] [Ideas]
使用ipynb，大家可以直接写代码并运行，更方便。

**Comment by Hzfengsy** [comment]
目前https://mlc.ai 中已经整合了ipynb，并且会在编译时运行。考虑到大家的修改需求，以后会考虑提供ipynb链接。同时想请问下使用colab对国内用户是否友好，或者有其他更好的notebook平台推荐吗

**Reply by hbsun2113** [reply]
您给的链接似乎是无效的？

**Reply by Hzfengsy** [reply]
抱歉，已更新

**Reply by zhiqwang** [reply]
可以用百度提供的 [AIStudio](https://aistudio.baidu.com/aistudio/index) 😄
